GMG/Scenes/Home/HomeView.swift

```diff
@@ -413,6 +230,173 @@ extension HomeView {
+                                router.push(.recording)
+                            },
+                            isExpanded: model.isScoresEmpty
+                        )
+                        
+                        ForEach(
+                            Array(model.recentScores.prefix(3).enumerated()),
+                            id: \.element.persistentModelID
+                        ) { (index, score) in
+                            ScoreCard(
+                                score: score,
+                                index: index,
+                                isSmall: true
+                            ) {
+                                router.push(
+                                    .chordProgress(score: score)
+                                )
+                            } renameScoreAction: { newTitle in
+                                intent.renameScore(score, newTitle: newTitle)
+                            } exportScoreAction: { score in
+                                router.push(.export)
+                            } deleteScoreAction: { score in
+                                intent.deleteScore(score, context: context)
+                            }
+                        }
+                    }
+                }
+                .frame(minHeight: 128)
+            }
+        }
+    }
+    
+    struct AddScoreButton: View {
+        let action: () -> Void
+        let isExpanded: Bool
+
+        var body: some View {
+            Button {
+                action()
+            } label: {
+                Image(systemName: "plus")
+                    .foregroundStyle(Color.black1)
+                    .frame(maxWidth: .infinity, maxHeight: .infinity)
+                    .background(
+                        Color.white,
+                        in: RoundedRectangle(cornerRadius: 18)
+                    )
+            }
+            .frame(width: isExpanded ? 156 : 77)
+        }
+    }
+    
+    struct AllFilesSection: View {
+        @Environment(\.modelContext) private var context
+        @Environment(Router.self) private var router: Router
+        
+        let model: HomeModelStateProtocol
+        let intent: HomeIntentProtocol
+        
+        var body: some View {
+            VStack(spacing: Spacing.md) {
+                HStack(alignment: .lastTextBaseline, spacing: 0) {
+                    Text("All Files")
+                        .font(Typography.WantedSansStd.R7)
+                        .foregroundStyle(Color.black1)
+                        .padding(.trailing, 20)
+
+                    Text("Latest")
+                        .font(Typography.WantedSansStd.R5)
+                        .foregroundStyle(
+                            model.isLatest ? Color.black5 : Color.black3
+                        )
+                        .padding(.trailing, 12)
+                        .onTapGesture { if model.isLatest == false { intent.toggleIsLatest() } }
```

Comment:
메서드 호출 의미를 명확하게 하는 것이 좋을 것 같습니다!

```suggestion
                        .onTapGesture { if model.isLatest == false { intent.setIsLatest(true) } }
```